#!/usr/bin/python3

def raise_exception_msg(message=""):
    """Raise a NameErroe exception with a message."""
    raise NameError(message)
